extensions/git/src/git.ts

```diff
@@ -1282,6 +1282,19 @@ export class Repository {
 		}
 	}
 
+	async blame(path: string): Promise<string> {
+		try {
+			const args = ['blame'];
+			args.push(path);
+
+			let result = await this.run(args);
+
+			return result.stdout.trim();
+		} catch (err) {
+			throw err;
```

Comment:
Leftovers? :smile:

Reply:
Nice catch, for now we only need to specificily handle one error type `no such path`.